guardian/gateway
# feat: Add more information to errors if we think cookies are missing

## What does this change?

If a user has cookies disabled the most likely error they'll encounter is a CSRF error as we expect this cookie to exist on almost every page visit and we check it before almost everything else.

<img width="451" alt="image" src="https://github.com/user-attachments/assets/69818904-06d4-4227-ac46-a35621334cba">

I spent some time trying to add safety checks to everywhere we check for cookies, but that became a mess of trying to propagate the correct errors to the correct places and figuring out where we expect cookies to sometimes not exist. I think this approach should catch the vast majority, if not all, of users that don't have cookies enabled.

## How to test

Turn off cookies and attempt a login, you should be redirected to a CSRF error page with the updated error message.
